resources/shared/params.mjs

```diff
@@ -146,21 +146,28 @@ class Params {
         return shuffleSeed;
     }
 
-    toSearchParams() {
+    toSearchParams(forRemote = false) {
         const rawParams = { ...this };
         rawParams["viewport"] = `${this.viewport.width}x${this.viewport.height}`;
+
+        // Only returning params that are useful for the workload.
+        // Both, 'suites' and 'tags', are specific to the debug menu.
+        if (forRemote) {
+            delete rawParams["suites"];
+            delete rawParams["tags"];
```

Comment:
I'd prefer having an allowed list of params instead of letting every param not listed here pass through.

Reply:
An allowlist is something we'd need to maintain, vs here we just remove what breaks the remote workload.

Reply:
we fixed the params obj without filtering keys.